### ISSUE TITLE
Highlighted English text displays incorrectly

### ISSUE THREAD

**Issue by artemith** [omnivore-app/logseq-omnivore#68]
The synced highlighted English text displays incorrectly like screenshot. The Chinese text displays correctly.

Logseq: 0.8.16
Omnivore plug-in: 1.11.2
OS: Windows 10

English:
![image](https://user-images.githubusercontent.com/14881525/213389894-c22eb8f0-903d-4a25-a628-8aeca44a2621.png)

Chinese:
![image](https://user-images.githubusercontent.com/14881525/213390370-6e14461c-2f06-4eea-82aa-fb811cb5cc41.png)

**Comment by sywhb** [comment]
Hey @artemith, thanks for reporting.

Do the other English articles also have the same issue? I'm not able to replicate it and it looks correct on my machine.
<img width="794" alt="Screenshot 2023-01-31 at 5 33 00 PM" src="https://user-images.githubusercontent.com/12963965/215722578-a9afe763-d95e-467e-bd6d-96036ad03086.png">

It looks like some formatting issue. Could you also try to resync Omnivore data and see?

**Comment by artemith** [comment]
> Hey @artemith, thanks for reporting.
> 
> Do the other English articles also have the same issue? I'm not able to replicate it and it looks correct on my machine. <img alt="Screenshot 2023-01-31 at 5 33 00 PM" width="794" src="https://user-images.githubusercontent.com/12963965/215722578-a9afe763-d95e-467e-bd6d-96036ad03086.png">
> 
> It looks like some formatting issue. Could you also try to resync Omnivore data and see?

Thanks for your reply! I just tested again. I found the articles collected by Microsoft Edge's Omnivore integration has this problem, whille those collected by Firefox's doesn't.

**Comment by sywhb** [comment]
@artemith Oh sorry, I didn't test on Edge. Let me try again

**Comment by sywhb** [comment]
Hey @jacksonh, do you have a Windows device that can be used to test on? I could not reproduce this issue on the Edge browser and the articles @artemith saved seem to be in a strange format

**Comment by sywhb** [comment]
Original HTML: https://gist.github.com/sywhb/6f72068117133f1ff1f4cb6cc1712c06

**Comment by sywhb** [comment]
> Hey @jacksonh, do you have a Windows device that can be used to test on? I could not reproduce this issue on the Edge browser and the articles @artemith saved seem to be in a strange format

Hey @jacksonh, could you help me with this one?

**Comment by artemith** [comment]
The bug does not appear in Omnivore plug-in 1.11.8 and Logseq 0.9.1.

**Comment by jacksonh** [comment]
@artemith thank you for following up on this one.

**Comment by artemith** [comment]
Thank you for your hard work!
